ee/features/permissions/components/dataroom-link-sheet.tsx

```
@@ -0,0 +1,898 @@
+    }
+  };
+
+  const createOrUpdateLinkWithPermissions = async (
+    linkData: DEFAULT_LINK_TYPE,
+    permissions: ItemPermission | null,
+    shouldPreview: boolean = false,
+    showSuccess: boolean = false,
+  ) => {
+    // Upload the image if it's a data URL
+    let blobUrl: string | null =
+      linkData.metaImage && linkData.metaImage.startsWith("data:")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)